website/pages/en/substreams.mdx

```diff
@@ -4,43 +4,36 @@ title: Substreams
 
 ![Substreams Logo](/img/substreams-logo.png)
 
-Substreams is a powerful blockchain indexing technology developed for The Graph Network. It enables developers to write Rust modules, compose data streams alongside the community, and provide extremely high-performance indexing due to parallelization in a streaming-first approach.
+Substreams is a powerful blockchain indexing technology developed to improve The Graph Network:
+- Substreams allows you to speed up the indexing times of your subgraph.
+- Substreams allows you to index non-EVM data (Solana, Injective, Starknet, Vara...)
 
-With Substreams, developers can quickly extract data from different blockchains (Ethereum, BNB, Solana, ect.) and send it to various locations of their choice, such as a Postgres database, a Mongo database, or a Subgraph. Additionally, Substreams packages enable developers to specify which data they want to extract from the blockchain.
+This is possible because your subgraph can use a Substreams package as a data source, thus creating a Substreams-powered Subgraph (SpS).
+A Substreams package a binary file that contains all the definitions of the data that you want to extract from the blockchain (similar to the `mapping.ts` file of your subgraph).
 
-## How Substreams Works in 4 Steps
+The following example shows a subgraph manifest (`subgraph.yaml`) that uses a Substreams package as data source, thus receiving the data extracted by the Substreams.
+Then, you can use this data in your `mapping.ts` and perform all the transformations needed as a usual subgraph.
```

Comment:
```suggestion
The following example shows a subgraph manifest (`subgraph.yaml`) that uses a Substreams package as data source, thus receiving the data extracted by the Substreams.

Then, you can use this data in your `mapping.ts` and perform all the transformations needed as a usual subgraph.
```

I'm thinking this is why the Prettier check is failing, but it could be more than one thing. If you're using VS Code, I recommend installing the Prettier extension with "auto-format on save" enabled.